README.md

```diff
@@ -32,9 +32,25 @@ The default logger and separator can be changed globally:
 RequestHandler.configure do
   logger Logger.new(STDERR)
   separator '____'
+  validation_backend RequestHandler::Validation::DryBackend
 end
 ```
 
+### Validation Backends
```

Comment:
can you change this (and in the code) to validation **engine**. feels like a better name then backend